config/initializers/permissions_policy.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+# Define an application-wide HTTP permissions policy. For further
+# information see https://developers.google.com/web/updates/2018/06/feature-policy
+#
+# Rails.application.config.permissions_policy do |f|
+#   f.camera      :none
+#   f.gyroscope   :none
+#   f.microphone  :none
+#   f.usb         :none
+#   f.fullscreen  :self
+#   f.payment     :self, "https://secure.example.com"
+# end
```

Comment:
Do we need a permission policy? If yes, then shouldn't we uncomment these, otherwise, why include the file?

Reply:
Sorry for the oversight. It was changed before I took over this Rails Upgrade, and I was not aware of this file.. Checking on the link provided over there, and also on Rails 7 Documentation, this permission_policy is merely recommended to up our security level but not required

And also for our use case, I think we don't need it for now, so we're safe to delete this file